packages/coinstac-ui/app/render/components/dashboard-home.jsx

```diff
@@ -23,16 +20,18 @@ class DashboardHome extends Component {
     };
   }
 
+/*
   componentWillMount() {
     const {
-      computations,
-      fetchRemoteResultsForUser,
+      // fetchRemoteResultsForUser,
```

Comment:
I understand things are rapidly changing, but I don't like committed commented out without a TODO if ya can

Reply:
I understand your concern. This is another instance of something that's not necessary and went undeleted after some testing.